typing/typetexp.mli

```diff
@@ -64,10 +64,15 @@ type error =
   | Unbound_class of Longident.t
   | Unbound_modtype of Longident.t
   | Unbound_cltype of Longident.t
-  | Ill_typed_functor_application of Longident.t
+  | Ill_typed_functor_application
+      of Longident.t * Longident.t * Includemod.error list option
```

Comment:
I have a small question here: is there a meaningful difference between `Some []` and `None` for the include error detail?

Reply:
I think neither is possible, so well, it's hard to say what counts as different.